File: cases/taint/go/argument-position-negative/argument_position_negative.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: argument-position-negative-input
	return 1
}

func chooseFirst(first int, second int) int { // DFB-WITNESS: argument-position-negative-first
	return first
}

func dfb_sink(value int) {} // DFB-SINK: argument-position-negative-sink

func run() {
	result := chooseFirst(0, dfb_source())
	dfb_sink(result)
}

File: cases/taint/kotlin/argument-position-negative/ArgumentPositionNegative.kt
package dataflowbench

object ArgumentPositionNegative {
    fun dfb_source(): Int { // DFB-SOURCE: argument-position-negative-input
        return 1
    }

    fun chooseFirst(first: Int, second: Int): Int { // DFB-WITNESS: argument-position-negative-first
        return first
    }

    fun dfb_sink(value: Int) {} // DFB-SINK: argument-position-negative-sink

    fun run() {
        val result = chooseFirst(0, dfb_source())
        dfb_sink(result)
    }
}
